src/tsf/Implementation.cpp

```diff
@@ -68,9 +68,11 @@ void Implementation::SetDefaultScopeAlphanumericHalfWidth(bool enable) noexcept
     s_wantsAnsiInputScope.store(enable, std::memory_order_relaxed);
 }
 
-void Implementation::Initialize()
+bool Implementation::Initialize()
 {
-    _categoryMgr = wil::CoCreateInstance<ITfCategoryMgr>(CLSID_TF_CategoryMgr, CLSCTX_INPROC_SERVER);
+    _categoryMgr = wil::CoCreateInstance<ITfCategoryMgr, wil::err_returncode_policy>(CLSID_TF_CategoryMgr, CLSCTX_INPROC_SERVER);
+    if (!_categoryMgr)
+        return false;

```

Comment:
Specifically, to match the rest of the file, it should look like this:
```suggestion
    if (!_categoryMgr)
    {
        return false;
    }

```